Drop const on returned values. Chromium code that includes these
headers builds with -Wignored-qualifiers.

Chromium commit c7faeda445225 (crbug.com/1095122, crbug.com/1212837).

diff --git a/src/opt/constant_polynomial.h b/src/opt/constant_polynomial.h
index f54e9856ddb06..e55061a846179 100644
--- a/src/opt/constant_polynomial.h
+++ b/src/opt/constant_polynomial.h
@@ -48,7 +48,7 @@ class ConstantPolynomial {
   }
 
   // Get the length.
-  const size_t Len() const { return coeffs_constant_.size(); }
+  size_t Len() const { return coeffs_constant_.size(); }
 
  private:
   // Private constructor.
diff --git a/src/symmetric_encryption.h b/src/symmetric_encryption.h
index b3631b2d2f753..31903252b1fb9 100644
--- a/src/symmetric_encryption.h
+++ b/src/symmetric_encryption.h
@@ -764,9 +764,9 @@ class SymmetricRlweKey {
   const typename ModularInt::Params* ModulusParams() const {
     return modulus_params_;
   }
-  const unsigned int BitsPerCoeff() const { return log_t_; }
-  const Uint64 Variance() const { return variance_; }
-  const unsigned int LogT() const { return log_t_; }
+  unsigned int BitsPerCoeff() const { return log_t_; }
+  Uint64 Variance() const { return variance_; }
+  unsigned int LogT() const { return log_t_; }
   const ModularInt& PlaintextModulus() const { return t_mod_; }
   const typename ModularInt::Params* PlaintextModulusParams() const {
     return plaintext_modulus_params_;
